docs/README-develop-node-utils-APIs.md

```diff
@@ -272,7 +303,7 @@ Example:
 
 As with the earlier example, `router eigrp` will need YAML definitions in the common file:
 
-`cisco_network_node_utils/lib/cisco_node_utils/command_reference_common.yaml`
+`cisco-network-node-utils/lib/cisco_node_utils/command_reference_common.yaml`
```

Comment:
just remove the prefix? `lib/cisco_node_utils/command_reference_common.yaml`. Same comment on next diff below.